ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -427,7 +427,32 @@ aws|aws-arm64|aws-usgov)
     ;;
 esac
 
-sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
+function setup-cvo() {
```

Comment:
nit: this function can get shifted up towards the top of the file to avoid distracting folks reading the main install portion and trying to understand the big-picture flow.  And a name like `set-cluster-version-spec-update-service` might be easier for folks skimming past the call site to understand what's happening.